students/mattmaeda/session02/grid_printer.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+
+def print_line(intersection_char, fill_char):
+    fill = "{} ".format(fill_char)
+    print(intersection_char, end=' ')
+    print(fill * 4, end='')
+    print(intersection_char, end=' ')
+    print(fill * 4, end='')
+    print(intersection_char)
+
```

Comment:
Good solution. Notice you are repeating some lines. How might you change your code to get rid of the repetition?